vrx_ign/src/SimpleBuoyancy.cc

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright (C) 2015 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
```

Comment:
Recommend adding  a note about the method used and the citation:
Erin Catto. “Exact Buoyancy for Polyhedra”. In Game Programming Gems 6, Charles River Media, 2006, pp. 175–187.

Reply:
Added to the header file in [19d536c](https://github.com/osrf/vrx/pull/500/commits/19d536c20c950b825ac3c5bb0ee970eebddbfdce).